editor/components/warning/style.scss

```diff
@@ -1,9 +1,9 @@
 .editor-warning {
 	z-index: z-index( '.editor-warning' );
 	position: absolute;
-	top: 50%;
+	top: 15px;
 	left: 50%;
-	transform: translate( -50%, -50% );
+	transform: translate( -50%, 0 );
```

Comment:
Minor: This can be changed to `translateX( -50% )` if we're not specifying a Y offset.